Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1490,19 +1498,48 @@ public string GetLocalApiUrl(IpAddressInfo ipAddress)
         {
             if (ipAddress.AddressFamily == IpAddressFamily.InterNetworkV6)
             {
-                return GetLocalApiUrl("[" + ipAddress.Address + "]");
+                return GetLocalApiUrlWithPort("[" + ipAddress.Address + "]");
             }
 
-            return GetLocalApiUrl(ipAddress.Address);
+            return GetLocalApiUrlWithPort(ipAddress.Address);
         }
 
-        public string GetLocalApiUrl(string host)
+        public string GetLocalApiUrlWithPort(string host)
         {
+            if (EnableHttps)
+            {
+                return string.Format("http://{0}:{1}",
```

Comment:
There's a bit of code duplication here, albeit minor. And this one should be https.
```suggestion
                return string.Format("https://{0}:{1}",
```